source/blog/articles/2019-06-fast-and-furious.html.markdown

```diff
@@ -0,0 +1,94 @@
+Kade, Cornelius, and Nelson stamp invites for the film showing.
+{: .caption-text .article-image__caption}
+
+[![Description](/assets/images/blog/2019/river-walk-700w.jpg)](/assets/images/blog/2019/river-walk-3000w.jpg)
+{: .article-image .article-image--has-caption}
+After a lot of walking in the heat, everyone enjoyed wading in the Dnister river!
+{: .caption-text .article-image__caption}
+
+Friday we had orientation all morning and the plan was that in the afternoon we were going to do surveys with people, then street preaching. Well, we got one out of the two. The surveys went great! We had four teams:
+
+* Team 1 Joshua, Ralph, and Abby Steele
+* Team 2 Nathan, Nelson, and Hosanna Steele
+* Team 3 Tolik and Corny
+* Team 4 Ben, Rebekah Steele, and I
+
+[![Description](/assets/images/blog/2019/preflight-for-surveys-700w.jpeg)](/assets/images/blog/2019/preflight-for-surveys-3000w.jpeg)
+{: .article-image .article-image--has-caption}
+A final briefing in the parking garage before we head out to do Bible surveys.
+{: .caption-text .article-image__caption}
+
+[![Description](/assets/images/blog/2019/survey-encounter-700w.jpg)](/assets/images/blog/2019/survey-encounter-3000w.jpg)
+{: .article-image .article-image--has-caption}
+Cornelius conducts Bible surveys in the center of L'viv.
+{: .caption-text .article-image__caption}
+
+Ben, Rebekah, and I's team was the "English team"  which means that we didn't have anyone on our team that could translate into Ukrainian. We had an absolute BLAST! I was surprised at how many people could speak English or if they couldn't, filled out the survey themselves. For each person that filled out a survey, we gave them a Good and Evil book and a starter packet to the Bible First correspondence course. The scariest thing that happened today was that Ben and I accidentally "stole" this girl's phone. This is how it happened…. While we were giving these two girls a survey, apparently during the exchange of us giving them Good and Evils and one of the girls handing back the clipboard, she accidentally handed over her phone as well. I thought it was Ben's phone and he thought it was mine. So a couple people later, after we had finished all our surveys, he handed me the phone and said, "Here's your phone." I said, "That's not my phone; I thought it was yours." Then we both looked at each other and were like "Oh NOOO!" So we started walking back trying to find the girls, and praise the Lord, the girl called her phone! Ben answered it, we were able to find her and give back the phone! It got a bit scary there for a minute though! Anyways, we had a good time laughing about it afterwards.  Next, we were going to do the street preaching and chalk drawing, but just exactly when we got everything set up it started to rain. And it kept raining. After sitting for about 30 mins in the rain with a tarp covering all the equipment it was decided that we would have to do the preaching Saturday. It was still a great day! 
```

Comment:
Kade said "today" in the fifth sentence of this paragraph. We could change it to "that day".

Reply:
Also we might want to delete a period after "This is how it happened".